ideas/315-core-improvements.md

```diff
@@ -0,0 +1,229 @@
+---
+id: #313-core-network-incentives
+title: Core Network Incentives Swarm Proposal
+status: research
+lead contributor: igor
+contributors:
+    - adam
+    - dmitry s
+    - roman
+    - pedro
+budget:
+- actual: xxx
+- estimate: yyy
+- currency: ETH/USD/SNT
+---
+
+# Core Networking Swarm proposal
+
+## Summary and Goal(s)
+
+Status Core Improvements swarm's goal is the preparation of Status Mobile Client to be released publicly (AppStore/GP release tracks).
+
+A few key areas that we need to focus on:
+
+* all the critical security and privacy issues are resolved (including audit);
```

Comment:
Additionally we should have a security and privacy artifact that:

- lists guarantees we make around security and privacy
- as well as ways we are falling short

This will inform security audit as well as branding/communication. Examples:

- Guarantee e2ee with forward secrecy
- guarantee minimal form of receive darkness, sender darkness so so due to x and y and heres what we do about it
- privacy etherscan tie ip to TX

etc.

@corpetty and I will help out with this as well.

Reply:
fixed